app/controllers/get_involved_controller.rb

```diff
@@ -46,6 +46,7 @@ def retrieve_special(params)
   def retrieve_date_filtered_closed_consultations(months)
     closed_count = 0
 
+    # Has to be a better way of doing this, will fail to count more than 500 results
```

Comment:
What happens if you remove the `per_page: 500`? Or set it to some extremely large number?

Reply:
The default value is 20 results per page when retrieving from the publishing api. This is all handled that end but it handles large numbers sensibly. If I remove it we don't get anywhere near the correct values because we don't have them all to count. The alternative is a bunch of looping logic doing it page by page with multiple calls to the API.

My concern is just how large a packet is being requested here. I could - and should - refine it into the loop for smaller calls, but if anything happens to the API calls in the loop who knows what will happen?